src/theme/index.ts

```diff
@@ -36,6 +37,7 @@ const theme = extendTheme({
    */
   components: {
     Button,
+    Tabs,
```

Comment:
Naming it the same as Chakra's own `Tabs`, allows the existing theme to be extended rather than overriding them.